pkg/webhook/controlplane/ensurer.go

```diff
@@ -203,9 +222,29 @@ func ensureKubeControllerManagerCommandLineArgs(c *corev1.Container, k8sVersion
 		c.Command = extensionswebhook.EnsureStringWithPrefixContains(c.Command, "--feature-gates=",
 			"InTreePluginGCEUnregister=true", ",")
 	}
-
 	c.Command = extensionswebhook.EnsureNoStringWithPrefix(c.Command, "--cloud-config=")
 	c.Command = extensionswebhook.EnsureNoStringWithPrefix(c.Command, "--external-cloud-volume-plugin=")
+	c.Command = extensionswebhook.EnsureNoStringWithPrefix(c.Command, "--allocate-node-cidrs=")
+	c.Command = append(c.Command, "--allocate-node-cidrs=false")
```

Comment:
We need this in case of IPv4 shoot clusters!

Reply:
This flag is always enabled in CCM.